### PR TITLE
Remove rinx from several yamls

### PR DESCRIPTION
**Pull request by rinx**
Signed-off-by: Rintaro Okamura <rintaro.okamura@gmail.com>

### Description:

- Remove me from chatops_permissions.yaml
- Remove me from dependabot.yaml

### Related Issue:

Nothing

### How Has This Been Tested?:

Nothing

### Environment:

- Go Version: 1.17.1
- Docker Version: 20.10.8
- Kubernetes Version: 1.22.0
- NGT Version: 1.12.3

### Types of changes:

- [ ] Bug fix [type/bug]
- [ ] New feature [type/feature]
- [ ] Add tests [type/test]
- [ ] Security related changes [type/security]
- [ ] Add documents [type/documentation]
- [ ] Refactoring [type/refactoring]
- [ ] Update dependencies [type/dependency]
- [ ] Update benchmarks and performances [type/bench]
- [X] Update CI [type/ci]

### Changes to Core Features:

- [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your core changes, as applicable?
- [ ] Have you successfully ran tests with your changes locally?

### Checklist:

- [X] I have read the [CONTRIBUTING](https://github.com/vdaas/vald/blob/master/CONTRIBUTING.md) document.
- [X] I have checked open [Pull Requests](https://github.com/vdaas/vald/pulls) for the similar feature or fixes?
- [ ] I have added tests and benchmarks to cover my changes.
- [ ] I have ensured all new and existing tests passed.
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: .github/chatops_permissions.yaml [hunk 1]
```diff
@@ -71,12 +71,6 @@ users:
     role:
       - author
       - admin
-  rinx:
-    role:
-      - author
```
Comment (kpango) [inline .github/chatops_permissions.yaml:76]: How about leaving it in the contributor role?

File: .github/dependabot.yml [hunk 2]
```diff
@@ -22,7 +22,7 @@ updates:
     open-pull-requests-limit: 10
     reviewers:
       - kpango
-      - rinx
```
Comment (kpango) [inline .github/dependabot.yml:25]: Can I add kmrmt?

**Review by kevindiu — DISMISSED** [review]

**Review by kpango — CHANGES_REQUESTED** [review]

**Review by hlts2 — COMMENTED** [review]
LGTM
after you add kmrmt account to dependabot.yml, I will approve this PR!

**Review by kpango — APPROVED** [review]
Your contribution to this project has been extremely valuable.
I believe in your decision and hope that your experience in this project will be useful to your future.

Thank you for the great job you did.

**Review by kevindiu — APPROVED** [review]
LGTM

**Comment by rinx** [comment]
Thank you all :wave:

**Comment by kpango** [comment]
I'll merge this PR tommorow.

**Pull request merged**